app/models/customer.py

```diff
@@ -1,4 +1,26 @@
 from app import db
+from datetime import datetime, timezone
 
 class Customer(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
\ No newline at end of file
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String)
+    registered_at = db.Column(db.DateTime)
+    postal_code = db.Column(db.String)
+    phone = db.Column(db.String)
+    
+    def to_json(self):
+        customer_dict = {
+                "id": self.id,
+                "name": self.name,
+                "registered_at": self.registered_at,
+                "postal_code": self.postal_code,
+                "phone": self.phone   
+            }
+        return customer_dict
+    
+    @classmethod
```

Comment:
ohhh love this! This is a very common built-in method that a lot of ORMs like sqlalchemy have that are a "create" method, which is what you have defined right here! Great idea!